OWNERS_ALIASES

```diff
@@ -170,7 +170,7 @@ aliases:
     - nckturner
     - wongma7
   provider-azure:
-    - craiglpeters
+    - andyzhangx
     - justaugustus
```

Comment:
Can be handled in a separate PR, but @justaugustus do you want to remain as an approver for azure?

Reply:
@mrbobbytables -- Feel free to remove me! This is vestigial from SIG Azure days.

Reply:
fixed, thanks folks for the confirm.